gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,136 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+    def create_fixed_significance_bin_lc_table(self, significance, significance_method, energy_range,
+                                               spectrum_extraction, separators=[]):
```

Comment:
Change `separators=[]` to `separators=None` and put the default in the method.
Read this: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html